elementary/monitor/cli.py

```diff
@@ -278,6 +288,7 @@ def monitor(
     suppression_interval,
     override_dbt_project_config,
     report_url,
+    filter,
```

Comment:
this is not very important but `filter` is a python builtin function and this means you are overriding it :\
can you set it to alerts_filter so it doesn't override? it can cause weird undebuggable bugs haha